api/src/main/java/org/eclipse/microprofile/reactive/messaging/Message.java

```diff
@@ -56,15 +56,12 @@ static <T> Message<T> of(T payload) {
      * Create a message with the given payload and metadata.
      * The acknowledgement and negative-acknowledgement are immediate.
      *
-     * @param payload The payload, must not be {@code null}.
+     * @param payload The payload.
      * @param metadata The metadata, if {@code null} an empty set of metadata is used.
```

Comment:
This code comes from the spec - we cannot change it.

Reply:
I know but I don't see any non-null requirement in the API code https://github.com/eclipse/microprofile-reactive-messaging/blob/master/api/src/main/java/org/eclipse/microprofile/reactive/messaging/Message.java

Reply:
Some would call this syndrome split-brain :D